python/composio/client/collections.py

```diff
@@ -1392,6 +1392,14 @@ def search_for_a_task(
             for task in response.json().get("items", [])
         ]
 
+    def list_enums(self) -> list[str]:
```

Comment:
Consider adding a docstring to the `list_enums` method explaining its purpose, return type, and any potential exceptions it might raise.